public/themes/default/css/editor/editor.css

```diff
@@ -30,7 +32,7 @@ html {
 
         &:not([show-top-level]) {
           --highlight-background-bw: #44ffd8;
-          dir-entry[path="."] {
+          dir-entry[path='.'] {
```

Comment:
These should stay `"`. I'll update the package.json to add in a `prettier:css` task because it looks like that's missing atm.